README.md

```diff
@@ -12,51 +12,135 @@ decrypted only by the controller running in the target cluster and
 
-```sh
-$ release=$(curl --silent "https://api.github.com/repos/bitnami-labs/sealed-secrets/releases/latest" | sed -n 's/.*"tag_name": *"\([^"]*\)".*/\1/p')
-
-# Install client-side tool into /usr/local/bin/
-$ GOOS=$(go env GOOS)
-$ GOARCH=$(go env GOARCH)
-$ wget https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/kubeseal-$GOOS-$GOARCH
-$ sudo install -m 755 kubeseal-$GOOS-$GOARCH /usr/local/bin/kubeseal
-
-# Note:  If installing on a GKE cluster, a ClusterRoleBinding may be needed to successfully deploy the controller in the final command.  Replace <your-email> with a valid email, and then deploy the cluster role binding:
-$ USER_EMAIL=<your-email>
-$ kubectl create clusterrolebinding $USER-cluster-admin-binding --clusterrole=cluster-admin --user=$USER_EMAIL
-
-# Install SealedSecret CRD, server-side controller into kube-system namespace (by default)
-# Note the second sealedsecret-crd.yaml file is not necessary for releases >= 0.8.0
-$ kubectl apply -f https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/controller.yaml
-$ kubectl apply -f https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/sealedsecret-crd.yaml
+* A cluster-side controller / operator
+* A client-side utility: `kubeseal`
+
+The `kubeseal` utility uses asymmetric crypto to encrypt secrets that only the controller can decrypt.
+
+These encrypted secrets are encoded in a `SealedSecret` resource, which you can see as a recipe for creating
+a secret. Here is how it looks like:
+
+```yaml
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  name: mysecret
+  namespace: mynamespace
+spec:
+  encryptedData:
+    foo: AgBy3i4OJSWK+PiTySYZZA9rO43cGDEq.....
 ```
 
-`controller.yaml` will create the `SealedSecret` resource and install the controller
-into `kube-system` namespace, create a service account and necessary
-RBAC roles.
+Once unsealed this will produce a secret equivalent to this:
 
-After a few moments, the controller will start, generate a key pair,
-and be ready for operation.  If it does not, check the controller
-logs.
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+data:
+  foo: bar  # <- base64 encoded "bar"
+```
+
+The [Usage](#usage) section explores in more detail how you craft SealedSecrets resources.
+
+### SealedSecrets as templates for secrets.
+
+The previous example only focused on the encrypted secret items themselves, but the relationship between a `SealedSecret` custom resource and the `Secret` it unseals into is similar in many ways (but not in all of them) to the familiar `Deployment` vs `Pod`.
+
+In particular, the annotations and labels of a `SealedSecret` resource, are not the same as the annotations of the `Secret` that gets generated out of it.
+
+To capture this distinction, the `SealedSecret` object has a `template` section which encodes all the fields want the controller uses when producing the unsealed `Secret`.
```

Comment:
"the fields want the controller uses" is weird, but I'm not quite sure what you meant to say.